packages/api/src/command/webhook/webhook.ts

```diff
@@ -195,12 +214,13 @@ export const sendPayload = async (
   }
 };
 
-export const sendTestPayload = async (url: string, key: string): Promise<boolean> => {
+export const sendTestPayload = async (url: string, key: string, cxId: string): Promise<boolean> => {
   const ping = nanoid();
   const payload: WebhookPingPayload = {
     ping,
   };
```

Comment:
I think we should add the meta to the ping message as well, this will allow users to also validate the ping message - an important step to validate their setup is working before running actual medical WH requests.

There's a ticket that mapped this a while ago, could update the ticket stating this is being done here:
https://github.com/metriport/metriport-internal/issues/898